adapters/Mintegral/MintegralAdapter/GADMediationAdapterMintegral.m

```diff
@@ -4,44 +4,100 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     http://www.apache.org/licenses/LICENSE-2.0
+//     https://www.apache.org/licenses/LICENSE-2.0
 //
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-//
 
 #import "GADMediationAdapterMintegral.h"
-
 #import "GADMediationAdapterMintegralConstants.h"
+#import "GADMAdapterMintegralExtras.h"
+#import "GADMAdapterMintegralRewardedAdLoader.h"
+#import "GADMAdapterMintegralBannerAdLoader.h"
+#import "GADMAdapterMintegralInterstitialAdLoader.h"
+#import "GADMAdapterMintegralNativeAdLoader.h"
+#import "GADMAdapterMintegralUtils.h"
 
-@implementation GADMediationAdapterMintegral
+#import <MTGSDK/MTGSDK.h>
+#import <MTGSDKBidding/MTGBiddingSDK.h>
 
-- (void)collectSignalsForRequestParameters:(nonnull GADRTBRequestParameters *)params
-                         completionHandler:
-                             (nonnull GADRTBSignalCompletionHandler)completionHandler {
-  // TODO: Start signal generation and call completionHandler when the signal generation finishes or
-  // fails.
-  completionHandler("", nil);
+
+@implementation GADMediationAdapterMintegral {
+    /// Mintegral rewarded ad.
+    GADMAdapterMintegralRewardedAdLoader * _rewardedAd;
+    
+    /// Mintegral banner ad.
+    GADMAdapterMintegralBannerAdLoader *_bannerAd;
+    
+    /// Mintegral interstitial ad.
+    GADMAdapterMintegralInterstitialAdLoader *_interstitialAd;
+    
+    /// Mintegral native ad.
+    GADMAdapterMintegralNativeAdLoader *_nativeAd;
```

Comment:
Nit, can you rearrange these variables so they are in alphabetical order?
```
    /// Mintegral banner ad.
    GADMAdapterMintegralBannerAdLoader *_bannerAd;

    /// Mintegral interstitial ad.
    GADMAdapterMintegralInterstitialAdLoader *_interstitialAd;

    /// Mintegral native ad.
    GADMAdapterMintegralNativeAdLoader *_nativeAd;

    /// Mintegral rewarded ad.
    GADMAdapterMintegralRewardedAdLoader * _rewardedAd;
```

Reply:
OK, I've made changes and committed.